can/generated_2021/uwrt_mars_rover_can_dump.txt

```diff
@@ -1,4 +1,4 @@
-================================= Messages =================================
+[?1049h[22;0;0t[1;19r(B[m[4l[?7h[19;1H[?1049l[23;0;0t
[?1l>================================= Messages =================================
```

Comment:
whats this mess of characters from? does it have something to do with the terminal config that the script was run in?